src/pages/Dashboard.js

```diff
@@ -12,28 +12,17 @@ import { GithubContext } from '../context/context';
 const Dashboard = () => {
   const { loading } = useContext(GithubContext);
 
-  if (loading) {
-    return (
-      <>
-        <Helmet>
-          <meta charset="utf-8" />
-          <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-          <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-          <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-          <meta
-            name="description"
-            content="Project for searching Github users and exploring their profiles"
-          />
-          <title>Github User</title>
-        </Helmet>
-        <main>
-          <Navbar />
-          <Search />
-          <img className='loading-img' src={loader} alt="loading information" />
-        </main>
-      </>
-    )
-  }
+  const ifLoadingContent = loading ? (
```

Comment:
It's better to give such names to boolean variable as `isLoading` `isActive` etc. 

Here's a good article on best practices for boolean variable naming

Reply:
Thanks. I decided to delete a variable and place the whole logic of rendering child components into component itself.